packages/client/src/schema/repository.ts

```diff
@@ -1039,11 +1440,17 @@ export const initObject = <T>(
           console.error(`Failed to parse link for field ${column.name}`);
         } else if (isObject(value)) {
           result[column.name] = initObject(db, schemaTables, linkTable, value);
+        } else {
+          result[column.name] = null;
         }
 
         break;
       }
       default:
+        result[column.name] = value ?? null;
+        if (column.notNull === true && value === null) {
+          console.error(`Parse error, column ${column.name} is non nullable and value resolves null`);
```

Comment:
I hope this doesn't happen, I considered throwing but wasn't sure. Made it safe in the end...

Reply:
👍 to make it safe.

I think at some point we'll need a "verbose/debug" option or an optional "log" function or something like that to decide how to print these messages. But of course, out of the scope of this PR. This is a very edge case and it's fine to leave it here for now.